backend/core/adapters/impl_file_converter.py

```diff
@@ -42,6 +45,10 @@ def find_priv(self, column: str):
         # finds the group with the most number of FPs
         fp_count = self.df[(self.df['is_fraud'] == 0) & (self.df['predicted_fraud'] == 1)].groupby(column).size().sort_values(ascending=True)
 
+        if fp_count.shape[0] != len(self.all_groups):
```

Comment:
actually this section of the code is bugged for main, it causes fail to upload file